packages/wdio-sauce-service/package.json

```diff
@@ -34,7 +34,7 @@
     "@wdio/types": "8.16.3",
     "@wdio/utils": "8.16.3",
     "ip": "^1.1.8",
-    "saucelabs": "7.2.2",
+    "saucelabs": "^7.3.1",
```

Comment:
This appears to have re-introduced the ESM loading problem
https://github.com/saucelabs/node-saucelabs/commit/a7814104cc324035901b9ade7d89c34182783175#commitcomment-127284025

Reply:
Thanks for letting us know, I will revert.